src/VSCodeUI.ts

```diff
@@ -53,6 +54,32 @@ export namespace VSCodeUI {
         }
     }
 
+    export function setJavaHomeIfRequired(terminal: Terminal): void {
+        const javaHome: string = workspace.getConfiguration("java").get<string>("home");
```

Comment:
this extension does not depend on `redhat.java`, it seems strange to read configuration like `java.home` from that extension. 😕